test/parallel_api/numeric/numeric.ops/adjacent_difference.pass.cpp

```diff
@@ -146,7 +146,7 @@ test(Pred pred)
 
     Sequence<T1> actual(max_len, [](::std::size_t i) { return T1(i); });
 
-    Sequence<T2> data(max_len, [value](::std::size_t i) { return i % 3 == 2 ? T2(i * i) : value; });
+    Sequence<T2> data(max_len, [&](::std::size_t i) { return i % 3 == 2 ? T2(i * i) : value; });
```

Comment:
I believe the same comment as in test/parallel_api/algorithm/alg.nonmodifying/find_end.pass.cpp

Reply:
Fixed.